credentials/alts/alts_test.go

```diff
@@ -128,9 +128,6 @@ func (s) TestInfo(t *testing.T) {
 	if got, want := info.SecurityProtocol, "alts"; got != want {
 		t.Errorf("info.SecurityProtocol=%v, want %v", got, want)
 	}
-	if got, want := info.SecurityVersion, "1.0"; got != want {
```

Comment:
Is there any harm in leaving this here?  If it's because of `vet.sh`, then:

1. why isn't `vet.sh` failing because of the place in the code that sets the value?
2. we can update `vet.sh` to ignore this deprecation warning instead.

Reply:
I left it in and added it to vet.sh.